dts/xtensa/intel/intel_adsp_ace20_lnl.dtsi

```diff
@@ -294,53 +294,49 @@
 			interrupt-parent = <&ace_intc>;
 		};
 
-		lps: lps@71ac0 {
-			compatible = "intel,adsp-lps";
-			reg = <0x00071ac0 0x100>;
+		dfpmcch: dfpmcch@71ac0 {
+			compatible = "intel,adsp-dfpmcch";
```

Comment:
what is dfpmcch? Can we use some human readable names?

Reply:
The dfpmcch device already defined in DTS bindings:
https://github.com/zephyrproject-rtos/zephyr/tree/main/dts/bindings/dfpmcch